beacon-chain/blockchain/service.go

```diff
@@ -200,27 +221,43 @@ func (c *ChainService) run(done <-chan struct{}) {
 			validatorsByHeight, err := c.chain.validatorsByHeightShard()
 			if err != nil {
 				log.Errorf("Unable to get validators by height and by shard: %v", err)
+				continue
 			}
-			log.Debugf("Received the following validators by height: %v", validatorsByHeight)
+			log.Debugf("Received the following number of validators by height: %v", len(validatorsByHeight))
```

Comment:
This is odd phrasing. Can you change this to `Received %d validators by height`?